src/librustc_resolve/resolve_imports.rs

```diff
@@ -973,7 +973,7 @@ impl<'a, 'b> ImportResolver<'a, 'b> {
                    max_vis.get() != ty::Visibility::Invisible && // Allow empty globs.
                    !max_vis.get().is_at_least(directive.vis.get(), &*self) {
                     let msg = "A non-empty glob must import something with the glob's visibility";
```

Comment:
```suggestion
                    let msg = "a non-empty glob must import something with the glob's visibility";
```

we usually make error messages lower-case :)

Reply:
In a comment @petrochenkov told we could write something along the lines of "this glob import doesn't import anything because all the candidates are too private", and I asked on "because no candidate is public enough". Should I change just the capital, or change the message to one of the two proposals ?